### PR TITLE
Refactor Kyuubi kubernetes block cleaner

### PR DESCRIPTION
**Pull request by ulysses-you**
### _Why are the changes needed?_

* fix bug as a blockMgr dir is created with no file at the begining.
* simplify code
* add test

### _How was this patch tested?_
the release directory structure
image

### CODE REVIEW

**Comment by ulysses-you** [comment]
cc @zwangsheng @yaooqinn

File: tools/kubernetes/spark-block-cleaner/src/test/scala/org.apache.kyuubi.tools/KubernetesSparkBlockCleanerTest.scala [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+package org.apache.kyuubi.tools
+
+import java.io.File
+import java.nio.file.{Files, Paths}
+import java.util.UUID
+
+import org.apache.kyuubi.KyuubiFunSuite
+
+class KubernetesSparkBlockCleanerTest extends KyuubiFunSuite {
```
Comment (yaooqinn) [inline tools/kubernetes/spark-block-cleaner/src/test/scala/org.apache.kyuubi.tools/KubernetesSparkBlockCleanerTest.scala:26]: KubernetesSparkBlockCleanerSuite for better regex matching

File: tools/kubernetes/spark-block-cleaner/src/test/scala/org.apache.kyuubi.tools/KubernetesSparkBlockCleanerTest.scala [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  import KubernetesSparkBlockCleanerConstants._
+
+  private val rootDir = System.getProperty("java.io.tmpdir")
```
Comment (yaooqinn) [inline tools/kubernetes/spark-block-cleaner/src/test/scala/org.apache.kyuubi.tools/KubernetesSparkBlockCleanerTest.scala:29]: can we use `Utils.createXXX`?

File: tools/kubernetes/spark-block-cleaner/src/main/scala/org/apache/kyuubi/tools/KubernetesSparkBlockCleaner.scala [hunk 3]
```diff
@@ -54,153 +56,142 @@ object KubernetesSparkBlockCleaner extends Logging {
...
+
+  private def checkConfiguration(): Unit = {
+    require(fileExpiredTime > 0,
+      s"the env $FILE_EXPIRED_TIME_KEY should be greater than 0")
+    require(deepCleanFileExpiredTime > 0,
+      s"the env $DEEP_CLEAN_FILE_EXPIRED_TIME_KEY should be greater than 0")
+    require(sleepTime > 0,
+      s"the env $SLEEP_TIME_KEY should be greater than 0")
+    require(freeSpaceThreshold > 0 && freeSpaceThreshold < 100,
+      s"the env $FREE_SPACE_THRESHOLD_KEY should between 0 and 100")
```
Comment (yaooqinn) [inline tools/kubernetes/spark-block-cleaner/src/main/scala/org/apache/kyuubi/tools/KubernetesSparkBlockCleaner.scala:77]: can we make these `kvs` more human-readable?

**Comment by yaooqinn** [comment]
can you fix the ci?

**Comment by ulysses-you** [comment]
see https://github.com/apache/incubator-kyuubi/pull/794

**Pull request closed without merging**